web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.ts

```diff
@@ -338,6 +338,7 @@ export default function useAccessRequestCheckout() {
 
     try {
       const { accessRequest } = await prepareAndCreateRequest({
+        reason: 'placeholder-reason',
```

Comment:
i think it's fine to remove this since we default it `prepareAndCreateRequest`

Reply:
I'll remove it from prepareAndCreateRequest to keep it consistent with teleport.e fix